openshift/machine-config-operator
# OCPBUGS-2079: Add ephemeral storage to kubelet system reserved args

Signed-off-by: Harshal Patil <harpatil@redhat.com>


Fixes: https://issues.redhat.com/browse/OCPBUGS-2079

**- What I did**
Add missing ephemeral storage value to system reserved arg of the kubelet

**- How to verify it**
Log into the node, and verify that kubelet process has started with the system reserved arg that has ephemeral storage

**- Description for the changelog**
